internal/writer/writer.go

```diff
@@ -66,7 +66,7 @@ func (b *Bundle) writeDockerfile(path string) error {
 	}
 	out += "\nCOPY manifests /manifests/\n"
 	out += "COPY metadata /metadata/\n"
-	return ioutil.WriteFile(path, []byte(out), os.ModePerm)
+	return ioutil.WriteFile(path, []byte(out), 0o664) // nolint:gosec
```

Comment:
```suggestion
	return ioutil.WriteFile(path, []byte(out), os.FileMode(664))
```

Would this be more clearer? It also seems to remove the gosec warning.

Reply:
SGTM, will update :+1:

Reply:
@muvaf you have to supply in octal notation :+1: See updated :)